packages/ui-components/src/icon/widgets/tabbarsvg.ts

```diff
@@ -61,7 +61,13 @@ export namespace TabBarSvg {
 
       return hpass(
         'div',
-        { className, title },
+        {
+          className,
+          title,
+          'aria-label': 'Close Launcher',
+          'aria-hidden': 'false',
+          role: 'button'
```

Comment:
I wonder if we should convert this div to a button rather than add the button role.